dashboard/app/controllers/courses_controller.rb

```diff
@@ -30,7 +30,7 @@ def new
   end
 
   def show
-    if current_user&.user_type == "teacher" && current_user.sections_instructed.any? {|s| s.course_id == @unit_group.id} && Experiment.enabled?(user: current_user, experiment_name: 'teacher-local-nav-v2')
+    if current_user&.user_type == "teacher" && current_user.sections_instructed.any? {|s| s.course_id == @unit_group.id} && (Experiment.enabled?(user: current_user, experiment_name: 'teacher-local-nav-v2') || DCDO.get('teacher_local_nav_v2', false))
```

Comment:
I'm pretty sure it is `teacher-local-nav-v2` with `-` instead of `_`

Reply:
OK, I copied this from sections_controller, so it looks like there's an error there as well.